### DISCUSSION TITLE
NSST in coupled model

### DISCUSSION THREAD

**Discussion by DeniseWorthen** [ufs-community/ufs-weather-model#3024] [General]
## Description

I'd like to understand the use of NSST in the coupled model. 

When coupled, the ocean model exports sea surface salinity, which is used solely by the ICE model. It is not imported to the ATM. The NSST model depends on the sea surface salinity. What value is it using and why isn't it using the actual ocean model salinity?

**Comment by yangfanglin** [comment]
@XuLi-NOAA Xu, can you help ?  It appears it is set to a constant in sfc_nst.f.

**Comment by XuLi-NOAA** [comment]
Denise,

Fanlin is right, the salinity is set to be a constant (34, I think) currently for the NSST model, the same as in the uncoupled mode.

The top layer salinity from the oceanic model should have been used, if it is imported to ATM, where the NSST model runs.

Relatively, the NSST model is not sensitive to salinity, but it is still good to have a more realistic value, if it is available.

Xu

**Comment by DeniseWorthen** [comment]
@XuLi-NOAA Thanks. What would be the current source for salinity? Is it annual WOA climatology? Or monthly perhaps?

Also, as I understand it, NSST was developed for the uncoupled model. I've always wondered about its use in a fully coupled model, where an prognostic ocean model is providing the surface temperature in response to the wind and other forcing. Was an evaluation done showing the need for for NSST in the coupled model ?

**Comment by yangfanglin** [comment]
@DeniseWorthen Back in 2021 significant effort had been made to assess the impact of NSST on coupled forecasts.  This [ppt ](https://docs.google.com/presentation/d/1jsMrY7_ixXGojFEANjzDI4G4FJc0IAl4/edit#slide=id.p7) was one of the few I can find.   The decision made at that time was to include NSST in the coupled model.   It was not clear how to handle NSST in data assimilation at that time.  I do not know if a decision has been made on the DA side.    Including more realistic salinity in NSST for computing density is desirable but probably won't have a large impact.

**Comment by DeniseWorthen** [comment]
@yangfanglin Thanks for the slides. I thought I had edited my comment about what salinity was used---it is constant either in standalone or coupled, I understand.

I'm not sure exactly how the NSST algorithm works, but I do know that at cooler/cold surface temperatures, salinity starts to determine the density. So if you only look at tropical SSTs, I agree you'd probably see no impact from using the real salinity.

**Comment by DeniseWorthen** [comment]
@XuLi-NOAA Do I understand correctly---once the ATM calculates NSST, it is used as the surface temperature over the ocean for the calculation of various fluxes (sensible etc)?

I'm thinking here of the case when we want to calculate the fluxes on the X-grid---in that case, if NSST is used in the fluxes, we'll need a method to calculate NSST on the X-grid, in the mediator. 

Is the algorithm for calculating NSST described anywhere (paper, ncep note etc)?
